app/motorTemperaturePublisher/conf/config.ini

```diff
@@ -1,5 +1,5 @@
 [GENERAL]
-robotname         ergocub
+robotname         ergocub2S
 period            1.0
-portprefix        /left_leg
-listofjoints      (0 1 2 3 4 5)
+portprefix        /upper_arm
+listofjoints      (0 1 )
```

Comment:
Hi @MSECode 

Out of sheer curiosity, what's `ergocub2S`?
Also, what do you identify as `upper_arm`? Two joints are normally not enough.

Reply:
just a random name I've already had for the upper_arm for ergocub 2 setup I'm using. 
For `upper_arm` we are identifying the full setup with 2 small joints and 2 medium, i.e. `shoulder_pitch`, `shoulder_roll`, `shoulder_yaw` and `elbow`. Nowever, we are currently working with just the small joints since the other 2 are disassembled.
I'll fix with the correct names as soon as the PR is ready and we have tested everything.